### DISCUSSION TITLE
V14. How can I see the zones that triggered an event.

### DISCUSSION THREAD

**Discussion by ediguidedog** [blakeblackshear/frigate#12055] [Ask A Question] [answered]
### What is your question:

In previous versions on the event page I can see a list of zones entered and the order they were entered. This is useful for automation debugging as often the mqtt message and trace are long gone by the time I get to it.  

Where in the v14 ui can I see this info?

**Comment by hawkeye217** [comment]
This is not currently available in 0.14 beta, but there’s a feature request here: https://github.com/blakeblackshear/frigate/issues/11399

**Reply by ediguidedog** [reply]
I'm not sure this covers it as it seems to be more about objects and bounding boxes.  What I would like to see is the information already available in earlier versions which has been removed. 

So in previous versions event page I see a list of zones and the order they were entered. I could then tailor notifications based on order.  So when I'm at home I could see someone arriving if they entered the path zone followed by the doorstep zone. On the way out they hit the doorstep zone first so there is no need for a notification.  This is really useful when for some reason I don't get a notification (or conversely do get one when someone is leaving) I can see at a glance the reason for this is zone related rather than any problem with the HA automation.

**Reply by NickM-27** [reply, marked as answer]
The feature request linked definitely fits this case exactly. Alerts for example are often setup to have required zones. Zones are one thing of many that you would check to see why an a review item was marked as alert or not 

And to be clear the notification behavior itself has not changed.

**Reply by ediguidedog** [reply]
Perfect thanks. I was aware nothing had changes (in this respect) message wise it was the the GUI
